src/nix/modules/composition/docker-compose.nix

```diff
@@ -63,6 +63,11 @@ in
       type = lib.types.attrsOf (lib.types.submodule service);
       description = "An attribute set of service configurations. A service specifies how to run an image as a container.";
     };
+    docker-compose.volumes = lib.mkOption {
+      type = lib.types.anything;
```

Comment:
The merge function of `anything` can result in surprising behavior. I'd find a submodule containing the bare minimum of options to be a more maintainable solution, as I won't have to guess what behaviors users actually rely on, when reviewing improvements later.

Reply:
Sounds good!

Reply:
Should i go with what is currently used for services?

https://github.com/KeepTruckin/arion/blob/issue-188/volumes-support/src/nix/modules/service/docker-compose-service.nix#L56-L60